lib/node/index.ts

```diff
@@ -37,12 +40,37 @@ function plugin(options: Options = {}): Plugin {
   const virtualModuleId = "virtual:remix-routes";
 
   const {
-    appDir = path.join(process.cwd(), "src"),
+    appDir: _appDir,
+    appDirectory = _appDir,
     importMode,
-    is404Route,
+    ...otherOptions
   } = options;
 
-  const prefix = appDir.replace(process.cwd(), "");
+  let dir = "";
+
+  // Check both src for backwards compatibility and app for remix's normal default
```

Comment:
I'm wondering if we should just rename `appDir` to `appDirectory` instead of making it backwards compatible.
We're still on v0 and there's _very_ little downloads at the moment.
I think mentioning it in the changelog would be sufficient.